debian/rules

```diff
@@ -26,7 +26,7 @@ extraopts += -DWITH_SYSTEMD=ON -DCEPH_SYSTEMD_ENV_DIR=/etc/default
 extraopts += -DWITH_GRAFANA=ON
 # assumes that ceph is exmpt from multiarch support, so we override the libdir.
 extraopts += -DCMAKE_INSTALL_LIBDIR=/usr/lib
-extraopts += -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib
+extraopts += -DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec
```

Comment:
also, i think we can drop this line.

see 

- https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
- https://github.com/Kitware/CMake/blob/60edc8344606acf7ef9327190d1fd66bd72bb865/Modules/GNUInstallDirs.cmake
- https://github.com/Debian/debhelper/blob/5d1bb29841043d8e47ebbdd043e6cd086cad508e/lib/Debian/Debhelper/Buildsystem/cmake.pm#L15

Reply:
If we want to clean that out later, perhaps, but given we're pretty explicit in `debian/rules`, I'll leave as-is.